ILRepack/Steps/AttributesRepackStep.cs

```diff
@@ -84,6 +84,14 @@ void CleanupAttributes()
             CleanupAttributes(typeof(SecurityCriticalAttribute).FullName, x => x.ConstructorArguments.Count == 0);
             CleanupAttributes(typeof(AllowPartiallyTrustedCallersAttribute).FullName, x => x.ConstructorArguments.Count == 0);
             CleanupAttributes(typeof(SecurityRulesAttribute).FullName, x => x.ConstructorArguments.Count == 0);
+            RemoveInternalsVisibleToAttribute();
+            RemoveAttributes<AssemblyDelaySignAttribute>(_ => true);
+            RemoveAttributes<AssemblyKeyFileAttribute>(_ => true);
+            RemoveAttributes<AssemblyKeyNameAttribute>(_ => true);
+        }
```

Comment:
I think the signing should still be removed, they can't be of any help (either they are wrong and do not match the signing opts given to ILRepack, or they match it and will be added anyway when signing).